WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsUnknownViewController.swift

```diff
@@ -405,7 +405,7 @@ private extension CardReaderSettingsUnknownViewController {
             comment: "Settings > Manage Card Reader > Connect > A button to begin a search for a reader"
         )
 
-        static var learnMore: NSAttributedString {
+        static var learnMore: NSAttributedString = {
```

Comment:
I think this can be an `static let` right?

Reply:
Yes, that's better 😁